DEVELOPMENT.md

```diff
@@ -118,3 +118,6 @@ organized by label.  `auto` recognizes the following PR labels:
   component; this is the default label for unlabelled PRs
 - `internal` — for changes only affecting the internal API
 - `documentation` — for changes only affecting the documentation
+- `tests` — for changes to tests
```

Comment:
we had `testing` with a single resolved issue #164, so I reassigned it to tests and removed testing label (I guess we could have just renamed testing into tests)